test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssert.cs

```diff
@@ -20,30 +22,62 @@ public static class XmlAssert
         /// <param name="actualXml">Actual xml string.</param>
         public static void Equal(string expectedXml, string actualXml)
         {
-            var sortedExpectedXDoc = SortAttributes(XDocument.Parse(expectedXml));
```

Comment:
I think we should not do this as we want to validate in case of invalid xmls being supplied(which would be missed if we had this check)..